tests/unit/testUtapiClient.js

```diff
@@ -9,18 +9,79 @@ const memoryBackend = new MemoryBackend();
 const ds = new Datastore();
 ds.setClient(memoryBackend);
 const REQUID = 'aaaaaaaaaaaaaaaaaaa';
-const bucket = 'foo';
-const newByteLength = 1024;
-
-// Set mock data of a particular size and count.
-function setData(objectSize, objectCount, timestamp) {
-    memoryBackend.data = {
-        's3:buckets:foo:storageUtilized:counter': objectSize,
-        's3:buckets:foo:numberOfObjects:counter': objectCount,
-        's3:buckets:foo:storageUtilized': [[timestamp, objectSize]],
-        's3:buckets:foo:numberOfObjects': [[timestamp, objectCount]],
-    };
-    return undefined;
+const metricTypes = {
+    bucket: 'foo-bucket',
+    accountId: 'foo-account',
+};
+
+// Get prefix values to construct the expected Redis schema keys
+function getPrefixValues(timestamp) {
+    const arr = [];
+    Object.keys(metricTypes).forEach(metric => {
+        if (metricTypes[metric] === undefined) {
+            return;
```

Comment:
for my understanding, you can't return from a `forEach` loop:
> There is no way to stop or break a forEach() loop other than by throwing an exception. If you need such behavior, the forEach() method is the wrong tool, use a plain loop instead. If you are testing the array elements for a predicate and need a Boolean return value, you can use every() or some() instead. If available, the new methods find() or findIndex() can be used for early termination upon true predicates as well.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach